Error during running web tests using microsoft edge browser

After running web tests with microsoft edge browser user's see provided below error

```java
Caused by: io.github.bonigarcia.wdm.config.WebDriverManagerException: java.lang.IllegalArgumentException: URI is not absolute
	at io.github.bonigarcia.wdm.managers.EdgeDriverManager.getDriversFromXml(EdgeDriverManager.java:308)
	at io.github.bonigarcia.wdm.managers.EdgeDriverManager.getDriverUrls(EdgeDriverManager.java:138)
	at io.github.bonigarcia.wdm.WebDriverManager.createUrlHandler(WebDriverManager.java:1468)
	at io.github.bonigarcia.wdm.WebDriverManager.download(WebDriverManager.java:1298)
	at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1204)
	at io.github.bonigarcia.wdm.WebDriverManager.fallback(WebDriverManager.java:1463)
	at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1441)
	at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1210)
	at io.github.bonigarcia.wdm.WebDriverManager.setup(WebDriverManager.java:429)
```

related issue in webdrivermanager library [link](https://github.com/bonigarcia/webdrivermanager/issues/1511)

--- a/engine/pom.xml
+++ b/engine/pom.xml
@@ -26,7 +26,7 @@
 
         <selenium-java.version>4.34.0</selenium-java.version>
         <appium-client.version>9.5.0</appium-client.version>
-        <webdrivermanager.version>6.2.0</webdrivermanager.version>
+        <webdrivermanager.version>6.3.2</webdrivermanager.version>
         <org.spring.boot.version>3.5.1</org.spring.boot.version>
         <org.spring.version>6.2.8</org.spring.version>
         <io.jsonwebtoken.version>0.12.6</io.jsonwebtoken.version>

